package.json

```diff
@@ -98,5 +104,6 @@
   },
   "publishConfig": {
     "access": "public"
-  }
+  },
+  "dependencies": {}
 }
```

Comment:
The test folder is not included in eslint check. 
There are quite a few lint errors once u include the folder inside

Reply:
how do you turn it on?